.circleci/src/workflows/harmony.yml

```diff
@@ -14,3 +14,10 @@ jobs:
       filters:
         branches:
           only: /^main$/
+  - harmony-deploy-chromatic:
+      requires:
+        - harmony-init
+      filters:
+        branches:
+          # not main
+          only: /^((?!main).)*$/
```

Comment:
We're gonna want to still run it on main as well, we just need to set it up on the chromatic UI so that it doesn't run "UI Tests" on main so that it just auto-approves.
But we def need it to run on main to become our snapshot to diff PRs against

Reply:
It's super confusing the way it's worded but basically we want to turn this guy off in order to essentially turn off comparing anything other than PRs->main
![image](https://github.com/AudiusProject/audius-protocol/assets/6711655/328608ed-bab8-4c55-a6fc-d14f37d9943c)

Reply:
Ah got it. So our CI builds will alert on PR when something changes, but none of them use their 'UI Review' so main will just become the baseline.

Does it know main is special and should become the baseline, or is it just that we're declining to review it so it always accepts the results?

Reply:
Oh wait I had it backwards, oops. We want `UI Review` to stay on, thats how we leave PR comparisons up, we want to turn off this guy. (again its a bit confusing haha)
![image](https://github.com/AudiusProject/audius-protocol/assets/6711655/a2b8a643-2156-473e-9ce8-f11c28d67740)

Reply:
> Does it know main is special and should become the baseline, or is it just that we're declining to review it so it always accepts the results?

I don't remember exactly how Chromatic has it set up. I cant remember which way it worked, but I do remember that there was no extra setup needed for it.
So basically once there are builds running in main and you raise a PR it should just work out of the box with no setup needed other than disabling that "Visual Tests" feature